app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,8 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"
+        android:maxSdkVersion="18"/>
```

Comment:
curious why are we restricting this permission to api 18 ?

Reply:
according to the [documentation](https://developer.android.com/reference/android/Manifest.permission#WRITE_EXTERNAL_STORAGE), from android API 19 this permission is implicitly granted to apps writing on their own directories and without that restriction you get warnings

Reply:
I see, it seems like we are saving the image in [Environment.getExternalStoragePublicDirectory]( https://github.com/dimagi/area-mapper/blob/master/app/src/main/java/richard/chard/lu/android/areamapper/SaveImageAsyncTask.java#L73) which is not an app specific but public directory and think we will need this permission to be able to make writes there. 

I think instead we want to set this to `28` instead as this permission doesn't do anything on api level 29 and above in the [scoped storage model](https://developer.android.com/training/data-storage#scoped-storage) which is enabled by default when we target android `30` or above. 

We should also test that that whether the image integration work between cc-android and area mapper. There is a related issue [here](https://dimagi-dev.atlassian.net/browse/SAAS-13203) that's proably happening due to scoped storage changes on api 30+.

Reply:
I missed that, thanks @shubham1g5. 
As per the issue with image capture, I'm planning to troubleshoot it once we have shifted to the new renderer. Let me know if you think I should look into it now.